src/stack.js

```diff
@@ -1,5 +1,16 @@
 class Stack {
-
+  constructor() {
+    this.stack = [];
+  }
+  push(el) {
+    return this.stack.unshift(el);
```

Comment:
Nice, I normally see students using `pop` and `push`. Shift and unshift work fine, its just completely backwards of how I conceptualize a stack.. but it behaves the same way.